graphql-samples/mutations/collaboration/create-relation-on-collaboration

```diff
@@ -0,0 +1,22 @@
+mutation createRelationOnCollaboration($data: CreateRelationOnCollaborationInput!){
```

Comment:
This should be under `graphql-samples/mutations/create/create-relation-on-collaboration`